keps/sig-node/3570-cpumanager/README.md

```diff
@@ -323,6 +407,236 @@ func (p *staticPolicy) RemoveContainer(s State, containerID string) error {
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+No to both.
+Changes in behavior only affects pods meeting the conditions (guaranteed QoS, integral CPU request) scheduler after the upgrade.
+Running pods will be unaffected by any change. This offers some degree of safety in both upgrade->rollback
+and upgrade->downgrade->upgrade scenarios.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No
+
+### Monitoring Requirements
+
+Monitor the pod admission counter
+Monitor the pods not going running after successful schedule
+
+###### How can an operator determine if the feature is in use by workloads?
+
+The operator need to inspect the node and verify the cpu pinning assignment either checking the cgroups on the node
+or accessing the podresources API of the kubelet.
+
+###### How can someone using this feature know that it is working for their instance?
+
+
+- [X] Other (treat as last resort)
+  - Details: the containers need to check the cpu set they are allowed to run; in addition, node agents (e.g. node_exporter)
+    can report the CPU assignment
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+- N/A
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [ ] Other (treat as last resort)
+  - Details:
+     a operator should check that pods go running correctly and the cpu pinning is performed. The latter can
+     be checked by inspecting the cgroups at node level.
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+No, because all the metrics we were aware of leaked hardware details.
+All of the metrics experimented by consumers of the feature so far require to expose hardware details of the
+worker nodes, and are dependent on the worker node hardware configuration (e.g. processor core layout).
+
+### Dependencies
+
+None
+
+###### Does this feature depend on any specific services running in the cluster?
+
+No
+
+### Scalability
+
+###### Will enabling / using this feature result in any new API calls?
+
+No, the feature is entirely node-local
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No, the feature is entirely node-local
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No, the feature is entirely node-local
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No, the feature is entirely node-local
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No, the feature is entirely node-local
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+No
```

Comment:
Seems useful to describe how resource usage could change in a give node when enabling this feature. Even though total amount of CPU time is the same, pinning CPUs would change resource usage is a non-neglible way right?

Reply:
That's a good point. I'll think and try to elaborate. Perhaps a third metric can help here.